packages/chaire-lib-backend/src/utils/processManagers/TrRoutingProcessManager.ts

```diff
@@ -81,15 +96,16 @@ const startTrRoutingProcess = async (
     return processStatus;
 };
 
-const start = async (parameters: { port?: number; debug?: boolean; cacheDirectoryPath?: string }) => {
+const start = async (parameters: StartParameters) => {
     const port = parameters.port || Preferences.get('trRouting.port');
     const cacheAllScenarios =
         config.trRoutingCacheAllScenarios === undefined ? false : config.trRoutingCacheAllScenarios;
 
     const params: Parameters<typeof startTrRoutingProcess>[3] = {
         debug: parameters.debug,
         cacheDirectoryPath: parameters.cacheDirectoryPath,
-        cacheAllScenarios: cacheAllScenarios
+        cacheAllScenarios: cacheAllScenarios,
+        logFiles: parameters.logFiles
```

Comment:
Not sure it's a good idea of expose the logFiles setting externally. We should know what we expect here.

Reply:
Another option for this PR would be to simply put the winston config in the config.js file (maybe not exactly the config, but something tha twill allow us to easily build that config). That would avoid all the parameter passing, each service can get its log configuration and we won't have to add too much when we want to support something else than log files.

Reply:
I updated so log files are only configurable through the config, there's no use case for individually setting it.